pkg/brew/tome.rb

```diff
@@ -0,0 +1,25 @@
+class Tome < Formula
+  HOMEPAGE = "https://github.com/toumorokoshi/tome"
+
+  desc "Modern replacement for 'sub'"
+  homepage HOMEPAGE
+  url HOMEPAGE
+  # TODO: What's correct license?
```

Comment:
@toumorokoshi Want me to add a license file to project, if so, what? I have a preference towards MIT or BSD if those are ones you approve.